File: be/server/src/main/java/com/baseball/domain/Batter.java
package com.baseball.domain;

public class Batter {
    private final String name;
    private final Integer plateAppearances;
    private final Integer hit;
    private final Integer out;
    private final Float average;

    public Batter(String name, Integer plateAppearances, Integer hit, Integer out, Float average) {
        this.name = name;
        this.plateAppearances = plateAppearances;
        this.hit = hit;
        this.out = out;
        this.average = average;
    }

    public String getName() {
        return name;
    }

    public Integer getPlateAppearances() {
        return plateAppearances;
    }

    public Integer getHit() {
        return hit;
    }

    public Integer getOut() {
        return out;
    }

    public Float getAverage() {
        return average;
    }
}

File: ios/baseball/baseball/Domain/Entity/Batter.swift
//
//  Batter.swift
//  baseball
//
//  Created by 이다훈 on 2021/05/07.
//

import Foundation

struct Batter: Codable {
    private (set) var name: String
    private (set) var plateAppearances: Int
    private (set) var hit: Int
    private (set) var out: Int
    private (set) var average: Float
}
